event/event.go

```diff
@@ -256,7 +256,10 @@ func (r *MessageSubscription) Serve(handler MessageHandler) error {
 					const size = 64 << 10
 					buf := make([]byte, size)
 					buf = buf[:runtime.Stack(buf, false)]
-					slog.Error("panic: message subscription: handling message", "message", msg, "stack_trace", string(buf))
+					slog.Error("panic: message subscription: handling message",
+						"error", err,
+						"message_body", msg.Body,
```

Comment:
Why not print the message with metadata too ?

Reply:
When I tried it using just `msg` as the value, it printed the message as a byte array. Example:

```
2024/04/19 12:56:40 ERROR panic: message subscription: handling message message="&{LoggableID:msg #0 Body:[123 34 116 114 97 99 101 95 105 100 34 58 34 34 44 34 111 114 103 97 110 105 122 97 116 105 111 110 95 105 100 34 58 34 34 44 34 110 97 109 101 34 58 34 104 101 108 108 111 34 44 34 101 118 101 110 116 34 58 123 34 116 101 120 116 34 58 34 66 111 110 106 111 117 114 34 125 125] Metadata:map[] BeforeSend:<nil> AfterSend:<nil> asFunc:0x851600 ack:0x82a600 nackable:true mu:{state:0 sema:0} isAcked:false}" stack_trace="goroutine 27 [running]:\ngithub.com/birdie-ai/golibs/event.(*MessageSubscription).Serve.func1.2()\n\t/home/levin/code/golibs/event/event.go:258 +0x59\npanic({0x8cdf00?, 0xdc73c0?})\n\t/usr/local/go/src/runtime/panic.go:770 +0x132\nmain.handler({0xc0003b23c0?, 0x4e?}, {{0xc000126245?, 0x0?}})\n\t/home/levin/code/foo/main.go:48 +0x7f\ngithub.com/birdie-ai/golibs/event.(*Subscription[...]).Serve.func1()\n\t/home/levin/code/golibs/event/event.go:169 +0x18a\ngithub.com/birdie-ai/golibs/event.(*Subscription[...]).Serve.SampledMessageHandler.func2()\n\t/home/levin/code/golibs/event/metrics.go:21 +0x11c\ngithub.com/birdie-ai/golibs/event.(*MessageSubscription).Serve.func1()\n\t/home/levin/code/golibs/event/event.go:266 +0x1a4\ncreated by github.com/birdie-ai/golibs/event.(*MessageSubscription).Serve in goroutine 9\n\t/home/levin/code/golibs/event/event.go:246 +0x47\n"
```

That's not super-useful :smile:

Reply:
ah damn..true...fucking byte arrays. Maybe log both on separate fields but forcing the body to be string